doc/nrf/ug_nrf9160_gs.rst

```diff
@@ -0,0 +1,502 @@
+.. _ug_nrf9160_gs:
+
+Getting started with nRF9160 DK
+###############################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This section will get you started with your nRF9160 DK.
+You will update the firmware (both the application firmware and the modem firmware) and the nRF Cloud certificates of the DK, and conduct some initial tests.
+
+If your DK  is v1.0.1 or lower, it is preprogrammed with the deprecated nRF9160: Asset Tracker application and you need to update the application firmware to :ref:`asset_tracker_v2`.
+The Asset Tracker v2 simulates sensor data and transmits it to Nordic Semiconductor's cloud solution, `nRF Cloud`_.
+
+Both the Asset Tracker v2 application and nRF Cloud are constantly being improved.
+Therefore, you should update the application and the modem firmware before you start working with the nRF9160 DK.
+
+.. _nrf9160_gs_requirements:
+
+Minimum requirements
+********************
+
+Make sure you have all the required hardware and that your computer has one of the supported operating systems.
+
+Hardware
+========
+
+* nRF9160 DK
+* nano-SIM card that supports :term:`LTE-M` or :term:`Narrowband Internet of Things (NB-IoT)` (the nRF9160 DK contains an iBasis SIM card)
+* micro-USB 2.0 cable
+
+Software
+========
+
+* Windows 8 or Windows 10
+* macOS
+* Linux
+
```

Comment:
The link to the nRF9160 DK Get started videos is missing.

Reply:
The video starts with telling the viewer to navigate to Infocenter, so I find it misleading at this point.

I also recall there being talk that the video does steps differently or in a different order, so linking to it as an example of the steps seems out of place.

Reply:
Since the video will be outdated, it should be communicated to the web team to remove it or create a new one.